bin/controller.py

```diff
@@ -4,12 +4,19 @@
 """
 
 import bottle as bt
+import re
 from pathlib import Path
 from metrics import Time
 from bin import root, config, models
 from bin.highlight import highlight, parse_language, parse_extension, languages
 
 
+BOTUARE = re.compile(r'|'.join([
+    re.escape('Mozilla/5.0 (compatible; Discordbot/2.0; +https://discordapp.com)'),
+    re.escape('facebookexternalhit/1.1 (+http://www.facebook.com/externalhit_uatext.php)'),
+]))
```

Comment:
@Mesteery what do you think ? We use `re.match` so the `^` and `$` are automatically injected

Reply:
The facebook one is from my server logs

Reply:
This looks good to me.

But why not just leave it as a list?
I didn't quite understand why we should go through a regex instead of a list, and in my opinion, going through a regex is a bit slower.

Reply:
Did you mean a set ? :D Matching a plaintext regexp is *ultra* fast ;)